pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -883,6 +896,31 @@ func endpointDiscoveryResponse(loadAssignments []*xdsapi.ClusterLoadAssignment,
 	return out
 }
 
+func endpointAcceptMtls(endpointTlsMap *map[string]bool, push *model.PushContext, ep *model.IstioEndpoint) bool {
+	b, err := json.Marshal(ep.Labels)
+	if err != nil {
+		return true
+	}
+	labelHash := fmt.Sprintf("%x", sha1.Sum(b))[:8]
```

Comment:
May be using sha256 instead?

Reply:
done. because seems sha1 can be dprecated, security collision is not so important given we are doing slice :8 and its for client side config

Reply:
https://support.servertastic.com/knowledgebase/article/deprecation-of-sha-1-and-moving-to-sha-2